mlir/lib/Dialect/MIOpen/Transforms/AffixTuningParameters.cpp

```diff
@@ -17,12 +17,15 @@ using namespace mlir;
 namespace {
 struct AffixTuningParameters : public MIOpenOpsAffixTuningParametersPassBase<AffixTuningParameters> {
 public:
-  AffixTuningParameters(LaunchDimensionCallback launchDimCallback)
-      : launchDimCallback(launchDimCallback) {}
+  AffixTuningParameters(int64_t blockSizeOverride,
+                        LaunchDimensionCallback launchDimCallback)
+      : launchDimCallback(launchDimCallback),
+        blockSizeOverride(blockSizeOverride) {}
   void runOnFunction() override;
 
 private:
   LaunchDimensionCallback launchDimCallback;
+  int64_t blockSizeOverride;
```

Comment:
the meaning of this field is better documented.

`blockSizeOverride` is not easy to comprehend.

Reply:
That's true, the block size is sort of set before and after the tuning process, so needs some good documentation on it. Added comments in the latest commit.